src/Microsoft.AspNetCore.Authentication.OpenIdConnect/Events/IOpenIdConnectEvents.cs

```diff
@@ -36,6 +36,11 @@ public interface IOpenIdConnectEvents : IRemoteAuthenticationEvents
         Task RedirectToIdentityProviderForSignOut(RedirectContext context);
 
         /// <summary>
+        /// Invoked before redirecting to the post logout redirect uri.
+        /// </summary>
+        Task RedirectToPostLogoutRedirectUri(RedirectContext context);
```

Comment:
It is a breaking change, cannot do it in 1.1. Can either do something clever, or can add a new interface.